charts/shoot-core/charts/csi-alicloud/templates/csi-diskplugin-ds.yaml

```diff
@@ -1,4 +1,108 @@
 {{- if .Values.enabled }}
+{{- if semverCompare ">= 1.13" .Values.kubernetesVersion -}}
```

Comment:
Why do we need this `if`-`else` block around the whole daemonset?!

Reply:
same reason as previous comment